README.md

```diff
@@ -15,18 +15,15 @@ Flexible and easy to use email for Elixir.
 
 - **Built-in support for popular mail delivery services**. Bamboo ships with
   [adapters for several popular mail delivery services, including Mandrill,
-  Mailgun, and SendGrid][available-adapters]. It's also quite easy to write
-  your own delivery adapter if your platform isn't yet supported.
-- **Deliver emails in the background**. Most of the time you don't want or need
-  to wait for the email to send. Bamboo makes it easy with
+  Mailgun, and SendGrid][available-adapters]. It's also quite easy to write your delivery adapter if your platform isn't yet supported.
+- **Deliver emails in the background**. Most of the time you don't want or need to wait for the email to send. Bamboo makes it easy with
   `Mailer.deliver_later`.
 - **A functional approach to mail delivery**. Emails are created, manipulated,
-  and sent using plain functions. This makes composition a breeze and fits
-  naturally into your existing Elixir app.
+  and sent using plain functions. This makes composition a breeze and fits naturally into your existing Elixir app.
 - **Unit test with ease**. Bamboo separates email creation and email delivery
   allowing you to test by asserting against email fields without the need for
   special functions.
-- **Dead-simple integration tests**. Bamboo provides helper functions to make
+- **dead-simple integration tests**. Bamboo provides helper functions to make
```

Comment:
I think this made more sense as it was, but I'm happy to change this if it makes more sense. Why did you think we should lower case the `d`?